contracts/MANACrowdsale.sol

```diff
@@ -94,26 +97,20 @@ contract MANACrowdsale is ContinuousCrowdsale, WhitelistedCrowdsale, CappedCrowd
         TokenPurchase(msg.sender, beneficiary, weiAmount, tokens);
 
         forwardFunds();
-
-        return tokens;
     }
 
     function setWallet(address _wallet) onlyOwner public {
         require(_wallet != 0x0);
         wallet = _wallet;
     }
 
-    function setRate(uint256 _rate) onlyOwner public {
-        require(_rate > 0);
+    function beginContinuousSale() onlyOwner public {
         require(isFinalized);
 
-        rate = _rate;
-        RateChange(_rate);
-    }
+        token.transferOwnership(continuousSale);
```

Comment:
At this point `continuousSale` is `0x0`. `transferOwnership` will fail silently in OpenZeppelin 1.2.0. It's fixed in master but not released yet. (FYI)